docs/cloud/grafana.md

```diff
@@ -0,0 +1,29 @@
+# Grafana
+
+GreptimeCloud service can be configured as a [Grafana data source](https://grafana.com/docs/grafana/latest/datasources/add-a-data-source/).
+
+## Prometheus
+
+Click the `Add data source` button and select `Prometheus` as the type.
+
+Fill in the following information in `HTTP`:
+
+- Prometheus server URL: `https://<host>/v1/prometheus/read?db=<dbname>`
+
+Click `basic auth` in the `Auth` section and fill in the following information in `Basic Auth Details`:
+
+- User: `<username>`
+- Password: *Your GreptimeCloud service password*
+
+Then click `Save & Test` button to test the connection.
+
+## MySQL
+
+Click the `Add data source` button and select `MySQL` as the type. Fill in the following information in `MySQL Connection`:
+
+- Host: `<host>:4002`
+- Database: `<dbname>`
+- User: `<username>`
+- Password: *Your GreptimeCloud service password*
```

Comment:
We need to teach user to set time zone to `UTC` as well. Because timestamp data type in mysql has no time zone information and grafana has no way to guess it.